composer.json

```
@@ -3,7 +3,7 @@
     "license": "MIT",
     "type": "project",
     "description": "Symfony Demo Application",
-    "minimum-stability": "stable",
+    "minimum-stability": "rc",
```

```suggestion
    "minimum-stability": "RC",
```

Apparently, the composer linter is picky about this. 🙈

---

Fortunately, it will be lowercase `stable` soon.

---

Fixed. Thanks.